server/graphql/v2/query/collection/KYCVerificationCollection.ts

```diff
@@ -0,0 +1,17 @@
+import { GraphQLList, GraphQLNonNull, GraphQLObjectType } from 'graphql';
+
+import { CollectionFields, GraphQLCollection } from '../../interface/Collection';
+import { GraphQLKYCVerification } from '../../object/KYCVerification';
+
+export const GraphQLKYCVerificationCollection = new GraphQLObjectType({
+  name: 'KYCVerificationCollection',
+  interfaces: [GraphQLCollection],
+  fields: () => {
+    return {
+      ...CollectionFields,
+      nodes: {
+        type: new GraphQLList(new GraphQLNonNull(GraphQLKYCVerification)),
+      },
+    };
+  },
+});
```

Comment:
This should actually be `server/graphql/v2/collection/KYCVerificationCollection.ts`, the query folder typically only contains the query resolver.